src/storekit.cs

```diff
@@ -469,5 +469,65 @@ interface SKCloudServiceController {
 		[Field ("SKCloudServiceCapabilitiesDidChangeNotification")]
 		NSString CloudServiceCapabilitiesDidChangeNotification { get; }
 	}
+
+	[NoTV, iOS (10,1)]
+	[BaseType (typeof(UIViewController))]
+	interface SKCloudServiceSetupViewController
+	{
+		// inlined ctor
+		[Export ("initWithNibName:bundle:")]
+		IntPtr Constructor ([NullAllowed] string nibName, [NullAllowed] NSBundle bundle);
```

Comment:
The test checks for a difference on how `init` (ObjC) and .ctor (C#) behave. OTOH it's not foolproof and this sounds (from the very few comments on the headers) like a OS VC (and one that you can't modify).

Please add a test (in monotouch-test) and, if you can't provide your own (like I suspect) then add a special case to the introspection tests.